### ISSUE TITLE
Is this library still maintained?

### ISSUE THREAD

**Issue by demetrio812** [teodorpatras/Jukebox#93]
Hello,
I see the library hasn't been maintained properly lately and I see a lot of fork and pull requests pending.

Are there any plan to maintain the library or has it been abandoned?

Cheers,
Dem

**Comment by demetrio812** [comment]
Clearly it’s not...

**Comment by JacobSyndeo** [comment]
Yeah, I'm shopping around for a replacement for another library I've used since 2012, but was seemingly abandoned by its author too. Bummer.
[AudioPlayer](https://github.com/delannoyk/AudioPlayer) looks promising, so I might choose that one. Still looking though!
